src/web/routes/router.ts

```diff
@@ -1,3 +1,7 @@
+// Must come before importing any instrumented module.
```

Comment:
**suggestion:** Consider moving this line to `/src/app.ts`, which represents an entry point to the application and is responsible for overall setup.

https://github.com/atlassian-labs/figma-for-jira/blob/341e806b610a5279120dc8faba7e8d1e9c24188a/src/app.ts#L1C1-L19C20